README.md

```diff
@@ -2,11 +2,11 @@
 
 An example app built using [Create React App](https://github.com/facebook/create-react-app) and the [DevCycle React SDK](https://docs.devcycle.com/sdk/client-side-sdks/react/)
 
-## Running the example
+## Getting Started
 
 * Run `yarn install` in the project directory to install dependencies
 * Create a `.env` file and set `REACT_APP_DEVCYCLE_CLIENT_SDK_KEY` to the SDK Key for your environment.\
-You can find this under [Settings > Environments](https://app.devcycle.com/r/environments) on the DevCycle dashboard.
+You can find this under [Settings > Environments](https://app.devcycle.com/r/environments) on the DevCycle dashboard. [Learn more about environments](https://docs.devcycle.com/essentials/environments).
 * Create a new feature on the dashboard with a string variable named `greeting` and a boolean variable named `spin-logo`
```

Comment:
should we link to the quickstart docs on creating a feature here?
https://docs.devcycle.com/introduction/quickstart#2-create-a-feature